extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -201,8 +197,14 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
         '"lock-bounds" requires "width" and "height" to be set on the amp-img child.'
       );
     }
-    // TODO(#31515): When aspect ratio is portrait, containerWidth will be used for this.
-    const percentScaledToFitViewport = containerHeight / ampImgHeight;
+
+    const containerRatio = containerWidth / containerHeight;
```

Comment:
Is this code assuming that `amp-story-panning-media` takes up the full page? I'm wondering about the cases where `panning-media`'s height would be, say, 50%, or when a `amp-story-grid-layer` has an `aspect-ratio`

Reply:
Nice! It's updated to measure the element instead of the page.